File: 0242-valid-anagram/0242-valid-anagram.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        if len(s) != len(t):
            return False

        
        s = list(s)
        t = list(t)
        
        for char in t:
            if not char in s:
                return False
            s.remove(char)
            
        return len(s) == 0

File: 0242-valid-anagram/0242-valid-anagram.java
class Solution {
    public boolean isAnagram(String s, String t) {
        int[] sMap = new int[26];
        int[] tMap = new int[26];
        
        for (char c : s.toCharArray()) {
            sMap[c - 'a']++;
        }
        
        for (char c : t.toCharArray()) {
            tMap[c - 'a']++;
        }
        
        return Arrays.equals(sMap, tMap);
    }
}